doc/cephfs/client-auth.rst

```diff
@@ -43,6 +43,25 @@ for example, to restrict client ``foo`` to ``mnt/bar`` directory, we will use. :
 
 ./ceph-fuse -n client.foo mnt -r /bar
 
+Free space reporting
+--------------------
+
+By default, when a client is mounting a sub-directory, the used space (``df``)
+will be calculated from the quota on that sub-directory, rather than reporting
+the overall amount of space used on the cluster.
+
+If you would like the client to report the overall usage of the filesystem,
+and not just the quota usage on the sub-directory mounted, then set the
+following config option on the client:
+
+::
+
+    client quota df = false
+
+If quotas are not enabled, or no quota is set on the sub-directory mounted,
+then the overall usage of the filesystem will be reported irrespective of
+the value of this setting.
```

Comment:
could we recursive to parent dir(if MDS cap allow us to ), to see if any quota is setted?
Otherwise it's actually a bug that user can mount a subdir (say /volume/tenantA/FeelFREE) to bypass quota enforced in /volume/tenantA.